ark/utils/test_utils.py

```diff
@@ -0,0 +1,168 @@
+import os
+import numpy as np
+import xarray as xr
+import skimage.io as io
+
+from mibidata import mibi_image as mi, tiff
+
+# required metadata for mibitiff writing (barf)
+METADATA = {
+    'run': '20180703_1234_test', 'date': '2017-09-16T15:26:00',
+    'coordinates': (12345, -67890), 'size': 500., 'slide': '857',
+    'fov_id': 'Point1', 'fov_name': 'R1C3_Tonsil',
+    'folder': 'Point1/RowNumber0/Depth_Profile0',
+    'dwell': 4, 'scans': '0,5', 'aperture': 'B',
+    'instrument': 'MIBIscope1', 'tissue': 'Tonsil',
+    'panel': '20170916_1x', 'mass_offset': 0.1, 'mass_gain': 0.2,
+    'time_resolution': 0.5, 'miscalibrated': False, 'check_reg': False,
+    'filename': '20180703_1234_test', 'description': 'test image',
+    'version': 'alpha',
+}
+
+
+def _gen_tif_data(fov_number, chan_number, img_shape, fills, dtype):
+    if fills is None:
+        tif_data = np.random.randint(0, 100,
+                                     fov_number * img_shape[0] * img_shape[1] * chan_number)
+        tif_data = tif_data.reshape((fov_number, *img_shape, chan_number)).astype(dtype)
```

Comment:
Any reason we can't use the size parameter for randint here? Seems like an extra line having to reshape it.

Reply:
👍

Reply:
TIL The `randint` `size` parameter actually refers to the result's `shape`, not its `size` lol